plugins/discord-rpc/back.js

```diff
@@ -0,0 +1,41 @@
+const DiscordRPC = require('discord-rpc');
+const rpc = new DiscordRPC.Client({
+	transport: 'ipc'
+});
+
+const clientId = '790655993809338398';
```

Comment:
Quick question about the ID, did you register an application in Discord under your own account?

Reply:
uhm yes, but it can be changed to yours if you want :)